src/plugins/event_annotation_listing/public/components/group_editor_flyout/group_preview.test.tsx

```diff
@@ -237,7 +237,7 @@ describe('group editor preview', () => {
     const assertTimeField = (fieldName: string, attributes: TypedLensByValueInput['attributes']) =>
       expect(
         (
-          attributes.state.datasourceStates.formBased.layers[DATA_LAYER_ID].columns[
+          attributes.state.datasourceStates?.formBased?.layers[DATA_LAYER_ID].columns[
```

Comment:
Why is even `datasourceStates` uncertain? 🙈

Reply:
Fixed :D